### PR TITLE
DDF-3224 added configurable list of context classes to idp handler

### PR DESCRIPTION
**Pull request by stustison**
#### What does this PR do?
 - fixed cardinality display of types other than text
 - added config drop down options for cardinality display in idp handler config
#### Who is reviewing it? 
@SmithJosh @garrettfreibott @tbatie 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@coyotesqrl
@stustison
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by stustison** [comment]
build now

File: platform/admin/ui/src/main/webapp/templates/configuration/textTypeList.handlebars [hunk 1]
```diff
@@ -12,7 +12,20 @@
...
+         {{/is}}
+         {{#isnt optionValues.length 0}}
+             <select name="value" class="form-control input-sm">
+                 {{#each optionValues}}
+                     <option value="{{.}}" {{#is this ../value}}selected="selected" {{/is}}>{{lookup ../optionLabels @index}}</option>
+                 {{/each}}
+             </select>
+         {{/isnt}}
+     {{else}}
+         <input type="{{#is type 1}}text{{/is}}{{#is type 5}}text{{/is}}{{#is type 6}}text{{/is}}{{#is type 7}}text{{/is}}{{#is type 8}}text{{/is}}{{#is type 9}}text{{/is}}{{#is type 10}}text{{/is}}{{#is type 12}}password{{/is}}{{#is type 2}}number{{/is}}{{#is type 3}}number{{/is}}{{#is type 4}}number{{/is}}" class="metatype" name="value" value="{{value}}">
```
Comment (coyotesqrl) [inline platform/admin/ui/src/main/webapp/templates/configuration/textTypeList.handlebars]: ✏️ A comment here would be nice to explain these magic numbers. Also, should you include an `else` condition?

File: platform/admin/ui/src/main/webapp/templates/configuration/textTypeList.handlebars [hunk 2]
```diff
@@ -12,7 +12,20 @@
  **/
  --}}
  <td>
-    <input type='text' name='value' class="{{id}} metatype" id='{{id}}' value='{{value}}'>
+     {{#if optionValues}}
+         {{#is optionValues.length 0}}
+             <input type="{{#is type 1}}text{{/is}}{{#is type 5}}text{{/is}}{{#is type 6}}text{{/is}}{{#is type 7}}text{{/is}}{{#is type 8}}text{{/is}}{{#is type 9}}text{{/is}}{{#is type 10}}text{{/is}}{{#is type 12}}password{{/is}}{{#is type 2}}number{{/is}}{{#is type 3}}number{{/is}}{{#is type 4}}number{{/is}}" class="metatype" name="value" value="{{value}}">
```
Comment (coyotesqrl) [inline platform/admin/ui/src/main/webapp/templates/configuration/textTypeList.handlebars]: ✏️ A comment here would be nice to explain these magic numbers. Also, should you include an else condition?

Comment (stustison) [inline platform/admin/ui/src/main/webapp/templates/configuration/textTypeList.handlebars]: It is invalid without a type, so it shouldn't need any sort of fallback (we don't currently have a fallback for the rest of the fields in the admin console either). I can add a comment though.

**Pull request merged**